core/src/main/java/org/apache/calcite/config/CalciteSystemProperty.java

```diff
@@ -214,6 +214,10 @@ public final class CalciteSystemProperty<T> {
         return ".";
       });
 
+
```

Comment:
Let's have them as `private static final` in `JdbcAdapterTest` for now, we can move them in a "common" location and adjust visibility if they are going to be used elsewhere

Reply:
Maybe we cannot put them in JdbcAdapterTest because need these static String in CalciteAssert (in Calcite testkit and JdbcAdapterTest is in core moudle ), how about put them in CalciteAssert with private static final ? @asolimando